### ISSUE TITLE
Blocks without transactions return array with empty string inside

### ISSUE THREAD

**Issue by jim-toth** [textury/arlocal#138]
`GET http://localhost:1984/block/height/0`
```
{
  "indep_hash": "4eqrcbdscydag0th2mgs4wn2d7q0w9k2x23k3mzh221xcwf6kwleakdi4nrb2v50",
  "timestamp": 1657744670,
  "previous_block": "",
  "height": 0,
  "txs": [
    ""
  ]
}
```

In the SQLite `blocks` table, this block has no transactions.  I think the fix here is to just return an empty array.

**Comment by jim-toth** [comment]
As block txIds are stored as a comma delimited string, the call to split is the source of the error in both block routes.

```typescript
''.split(',') // results in ['']
```

**Comment by cedriking** [comment]
Thanks @jim-toth for noticing this. Fix has been released on version `1.1.58`.
